app/scripts/phishing-detect.js

```diff
@@ -11,8 +11,11 @@ document.addEventListener('DOMContentLoaded', start);
 function start() {
   const hash = window.location.hash.substring(1);
   const suspect = querystring.parse(hash);
+  const newIssueLink = document.getElementById('new-issue-link');
+  const csdbLink = document.getElementById('csdbLink');
 
-  document.getElementById('csdbLink').href = `https://cryptoscamdb.org/search`;
+  csdbLink.href = `https://cryptoscamdb.org/search`;
+  newIssueLink.href = `https://github.com/MetaMask/eth-phishing-detect/issues/new?title=[Legitimate%20Site%20Blocked]%20${suspect.hostname}&body=${suspect.href}`;
```

Comment:
We should escape the hostname and href here, just to be safe. I'm not sure whether anything screwy can happen here since the character set allowed in hostnames is limited, but why take chances. It is untrusted user input after all.

Reply:
good call, done!